### PR TITLE
You can now point while handcuffed (No one can yell at me edition)

### PR DESCRIPTION
**Pull request by azzzertyy**
# Document the changes in your pull request
as title

# Why is this good for the game?
small quality of life change which makes sense because cuffs dont magically prevent you from pressing E and typing "points at the airlock"
# Changelog
:cl:  
tweak: You can now point while handcuffed
/:cl:

### CODE REVIEW

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -367,7 +367,10 @@
 
 //same as above
 /mob/living/pointed(atom/A as mob|obj|turf in view())
-	if(incapacitated())
+	var/obj/item/clothing/suit/straightjacket = src.get_item_by_slot(ITEM_SLOT_OCLOTHING)
```
Comment (iamthesilly) [inline code/modules/mob/living/living.dm]:
```suggestion
	var/obj/item/clothing/suit/straightjacket/straightjacket = src.get_item_by_slot(ITEM_SLOT_OCLOTHING)
```

**Review by SapphicOverload — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -367,7 +367,10 @@
 
 //same as above
 /mob/living/pointed(atom/A as mob|obj|turf in view())
-	if(incapacitated())
+	var/obj/item/clothing/suit/straightjacket/straightjacket = src.get_item_by_slot(ITEM_SLOT_OCLOTHING)
```
Comment (iamthesilly) [inline code/modules/mob/living/living.dm]: My bad, spelling error
```suggestion
	var/obj/item/clothing/suit/straight_jacket/straightjacket = src.get_item_by_slot(ITEM_SLOT_OCLOTHING)
```

**Review by SapphicOverload — DISMISSED** [review]

**Comment by Yarinoi** [comment]
ggrgrgrgr HOW DARE YOU this gives a NEW AVENUE FOR PRISONERS TO MESS WITH SECURITY this will destroy sec you MONSTER WHAT NEXT ARE PERMA PRISONERS GOING TO HAVE AN ARMORY FULL OF PULSE RIFLES IN PERMA

**Comment by azzzertyy** [comment]
> ggrgrgrgr HOW DARE YOU this gives a NEW AVENUE FOR PRISONERS TO MESS WITH SECURITY this will destroy sec you MONSTER WHAT NEXT ARE PERMA PRISONERS GOING TO HAVE AN ARMORY FULL OF PULSE RIFLES IN PERMA

no yelling! read the title! no yelling at me edition!

**Comment by Yarinoi** [comment]
> > ggrgrgrgr HOW DARE YOU this gives a NEW AVENUE FOR PRISONERS TO MESS WITH SECURITY this will destroy sec you MONSTER WHAT NEXT ARE PERMA PRISONERS GOING TO HAVE AN ARMORY FULL OF PULSE RIFLES IN PERMA
> 
> no yelling! read the title! no yelling at me edition!

oh ok sorry

ᵍᵍʳᵍʳᵍʳᵍʳ ʰᵒʷ ᵈᵃʳᵉ ʸᵒᵘ ᵗʰᶦˢ ᵍᶦᵛᵉˢ ᵃ ⁿᵉʷ ᵃᵛᵉⁿᵘᵉ ᶠᵒʳ ᵖʳᶦˢᵒⁿᵉʳˢ ᵗᵒ ᵐᵉˢˢ ʷᶦᵗʰ ˢᵉᶜᵘʳᶦᵗʸ ᵗʰᶦˢ ʷᶦˡˡ ᵈᵉˢᵗʳᵒʸ ˢᵉᶜ ʸᵒᵘ ᵐᵒⁿˢᵗᵉʳ ʷʰᵃᵗ ⁿᵉˣᵗ ᵃʳᵉ ᵖᵉʳᵐᵃ ᵖʳᶦˢᵒⁿᵉʳˢ ᵍᵒᶦⁿᵍ ᵗᵒ ʰᵃᵛᵉ ᵃⁿ ᵃʳᵐᵒʳʸ ᶠᵘˡˡ ᵒᶠ ᵖᵘˡˢᵉ ʳᶦᶠˡᵉˢ ᶦⁿ ᵖᵉʳᵐᵃ

**Comment by MajManatee** [comment]
nice

**Comment by Moja10** [comment]
I agree, prisoners should be able to point, even if the handcuffs are behind them
image

**Comment by MajManatee** [comment]
I work in a prison and I can confirm that people can use their fingers and arms while cuffed

**Comment by SomeguyManperson** [comment]
Only people this will benefit are people who have managed to get handcuffed, with the added bonus that it is "countered" by something that will get you as much flak as handcuffed shoes (which a substantial amount of people want removed)

I also don't see why we want to make not killing people as an antagonist harder

**Comment by Yarinoi** [comment]
if you ask me, i dont see how pointing is any more annoying then prisoners constantly hurling the worst things that are still allowed by the server at anyone around them, plus pointing is a decently useful tool

**Comment by MajManatee** [comment]
option to alt click cuffs to allow pointing by "adjusting their fit" before cuffing someone? Or better yet having point mode be the default but being switchable to point stopping.

**Comment by SomeguyManperson** [comment]
> if you ask me, i dont see how pointing is any more annoying then prisoners constantly hurling the worst things that are still allowed by the server at anyone around them,

this primarily effects mimes in terms of security, it's more harmful to antagonists who have to/want to transport a non-dead and non-unconscious person since pointing is fast, easy, and nearly impossible to prevent

> plus pointing is a decently useful tool

cuffs exist to remove tools from people. They also prevent opening access-locked doors (so someone with access to an area but is cuffed can't simply run through a few doors), and prevent climbing over tables (there IS actually framing for climbing tables while cuffed, with it penalizing your climbing speed, it just doesn't work since you can't do any channels period)

What this does is give handcuffed people a useful tool which is even harder to remove than other tools they retain i.e. speech which is countered by mute toxin and a myriad of antagonist abilities. Your only option is the strait jacket, which is awful. And no, I tested, mime's bane does not work on pointing

**Comment by ToasterBiome** [comment]
Now draw her being tested

File: code/modules/mob/living/living.dm [hunk 3]
```diff
@@ -367,7 +367,10 @@
 
 //same as above
 /mob/living/pointed(atom/A as mob|obj|turf in view())
-	if(incapacitated())
+	var/obj/item/clothing/suit/straight_jacket/straightjacket = src.get_item_by_slot(ITEM_SLOT_OCLOTHING)
```
Comment (JohnFulpWillard) [inline code/modules/mob/living/living.dm]:
```suggestion
	var/obj/item/clothing/suit/straight_jacket/straightjacket = get_item_by_slot(ITEM_SLOT_OCLOTHING)
```

**Comment by MajManatee** [comment]
No activity. Reopen when work continues

**Review by JohnFulpWillard — APPROVED** [review]

**Pull request merged**